### ISSUE TITLE
Linux install script should support hardened environments

### ISSUE THREAD

**Issue by janosdebugs** [opentofu/get.opentofu.org#15]
See opentofu/opentofu#1126

**Comment by Kazzhar** [comment]
Hi, I would like to work on this. Could you please assign this to me?

**Comment by janosdebugs** [comment]
Hi @Kazzhar thank you for volunteering, I've assigned you. Please read [the "I've been assigned an issue, now what?" section of the contribution guide](https://github.com/opentofu/opentofu/blob/main/CONTRIBUTING.md#ive-been-assigned-an-issue-now-what) before you begin working.

**Comment by Kazzhar** [comment]
thanks!

**Comment by Kazzhar** [comment]
I want to point out something I noticed. The changes in deb-step2.sh [here](https://github.com/opentofu/opentofu/blob/main/website/docs/intro/install/deb-step2.sh) and in deb-step3.sh [here](https://github.com/opentofu/opentofu/blob/main/website/docs/intro/install/deb-step3.sh) are not reflected on the [website docs](https://opentofu.org/docs/intro/install/deb/). 

Not sure if this is intentional and there's a perfectly normal explanation for it that I'm missing out on, haha.

**Comment by janosdebugs** [comment]
@Kazzhar we update the website semi-regularly as it currently involves manual work. (We should fix that.)

**Comment by Kazzhar** [comment]
Is there an issue open for this? I'm curious to how this could be worked on.

**Comment by janosdebugs** [comment]
Not really, it's basically just a PR to the `opentofu.org` repo that updates the `opentofu-repo` folder (submodule) to the latest v1.6 branch. This assumes the change has been backported to the v1.6 branch, which it may not have been.

If you'd like to get this deployed, feel free to submit the PR(s), otherwise I'll take a look later. There's a "backporting" section in the contrib guide in the main repo.
